ext/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+#######################
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+#  agreements.  See the NOTICE file distributed with this work for additional information regarding
+#  copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with the License.  You may obtain
+#  a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing permissions and limitations under
+#  the License.
+#
+#######################
+
+#
+# This directory can be used to extend the ATS cmake build to include additional plugins or cripts
+# directly in the ATS build
+#
+
+include(add_atsplugin)
+include(add_cripts_bundle)
+
+file(
+  GLOB EXT_SUBDIRS
+  LIST_DIRECTORIES true
+  *
+)
+
+foreach(SDIR ${EXT_SUBDIRS})
+  if(IS_DIRECTORY ${SDIR})
+    add_subdirectory(${SDIR})
```

Comment:
What if the directory doesn't contain a CMakeLists.txt?

Reply:
I can check for that!  I guess they can be ignored, but maybe I'll print a warning